OM-3006: changing default for pagination (#18)
OnCorps/doctrine2-exporter: README_ApiPlatform.md

    Allows management of pagination. Options:
    

>     enabled: [true | false] - If option is fully omitted then default would be hinerited by global configuration in api-platform.yaml

Reviewer: ```suggestion
    enabled: [true | false] - If not specified then the default is to inherit the global setting in api-platform.yaml
```
